benchmark/DriverBench/SingleDocBench.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace MongoDB\Benchmark\DriverBench;
+
+use Generator;
+use MongoDB\Benchmark\BaseBench;
+use PhpBench\Attributes as Bench;
+
+use function array_map;
+use function range;
+
+/** @see https://github.com/mongodb/specifications/blob/ddfc8b583d49aaf8c4c19fa01255afb66b36b92e/source/benchmarking/benchmarking.rst#single-doc-benchmarks */
+#[Bench\BeforeMethods('prepareDatabase')]
+final class SingleDocBench extends BaseBench
+{
+    public function prepareDatabase(): void
+    {
+        self::getCollection()->drop();
+    }
+
+    public function setupFindOneByID(): void
+    {
+        $tweet = (array) self::readJsonFile(self::TWEET_FILE_PATH);
+        $docs = array_map(fn ($id) => $tweet + ['_id' => $id], range(1, 10_000));
+        self::getCollection()->insertMany($docs);
```

Comment:
> For each of the 10,000 sequential `_id` numbers, issue a find command for that _id on the 'corpus' collection and retrieve the single-document result.

As we can see in the "small document insert one" benchmark, inserting one-by-one is very slow. That's why I used `insertMany`. That should not matter as this is a setup operation out of the performance measurements.